js/storage.js

```diff
@@ -9,22 +9,22 @@ function getStorageTasks() {
     }
```

Comment:
you can create `Store` class in OOP style and use it for tasks, serialization, desirialization and etc

Reply:
thanks, I created it
```suggestion
    export default class Storage { ... }
```